app/main.py

```diff
@@ -202,14 +202,15 @@ def rabbit_status_callback(self, response):
             status = res.get('status')
             logger.info("Rabbit MQ health check response {}".format(status))
             if status == "ok":
-                logger.info('Setting status now')
+                logger.info('Rabbit MQ health ok')
                 self.rabbit_status = True
 
     def determine_ftp_status(self):
         try:
             self.ftp_status = False
             conn = self.ftp.get_connection()
             if conn:
+                logger.info('FTP health ok')
                 self.ftp_status = True
         except Error as e:
             logger.error("FTP error raised", error=str(e))
```

Comment:
Surely if an exception is occuring then it's not an "error" per se, it's a report of bad health?

Reply:
The ftp library used raises any issues with connection as an error. Should we handle this as an exception? Then use info/debug log level

Reply:
If you change the import to `from ftplib import Error as FTPException` this would be more explicit. At the same time, could you reorder the imports so they comply with our standards?